lib/engine/game/base.rb

```diff
@@ -192,6 +192,23 @@ def initialize(names, id: 0, actions: [])
         @names = names.freeze
         @players = @names.map { |name| Player.new(name) }
 
+        case self.class::DEV_STAGE
+        when :prealpha
+          @log << "#{self.class.title} is in prealpha state, no support is provided at all"
+        when :alpha
+          @log << "#{self.class.title} is currently considered 'alpha',"\
+          ' the rules implementation is likely to not be complete.'
+          @log << 'As the implementation improves, games that are not compatible'\
+          ' with the latest version will be deleted.'
+          @log << 'We suggest that any alpha quality game is concluded within 7 days.'
+        when :beta
+          @log << "#{self.class.title} is currently considered 'beta',"\
+          ' the rules implementation may allow illegal moves.'
+          @log << 'As the implementation improves, games that are not compatible'\
+          ' with the latest version will be given 7 days to be completed before being deleted.'
+          @log << 'Because of this we suggest not playing games that may take months to complete.'
```

Comment:
but with your opal changes will this be true anymore?

Reply:
Depends on how often we want to pin, are we gonna keep them all?

Reply:
yea maybe not

Reply:
it's probably easier to give the users a "you have 7 days" rather than leaving them with the impression that they might have infinity time to complete.
We may of course be more generous than that :)